language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ObjectBodiedNodeContextProvider.java

```diff
@@ -67,6 +70,9 @@ public ObjectBodiedNodeContextProvider(Class<T> attachmentPoint) {
             completionItems.add(new SnippetCompletionItem(context, Snippet.KW_RESOURCE.get()));
             completionItems.add(new SnippetCompletionItem(context, Snippet.DEF_REMOTE_FUNCTION.get()));
             completionItems.add(new SnippetCompletionItem(context, Snippet.DEF_RESOURCE_FUNCTION_SIGNATURE.get()));
+            if (this.onSuggestInitFunction(node)) {
```

Comment:
Shall we rename this to `onSuggestInitMethod(node)` as we are trying to suggest a `method`